src/lambda.js

```diff
@@ -0,0 +1,135 @@
+const alerts = require("./alerts.js");
```

Comment:
from a testing/readability perspective, might be good to break this file into some smaller functions that can be tested individually